react/src/pages/participants/AnalysisRunnerDialog.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, ChangeEvent } from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { TransitionProps } from '@material-ui/core/transitions';
+import { Participant } from './ParticipantsTable';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+import Typography from '@material-ui/core/Typography';
+import Checkbox, { CheckboxProps } from '@material-ui/core/Checkbox';
+import DatasetTable from '../analysis/DatasetTable';
+import { annotations } from '../analysis/AnalysisInfoDialog';
+import ChipStrip from '../analysis/ChipStrip';
+
+interface CancelAnalysisDialogProp {
+  participants: Participant[],
+  open: boolean,
+  onClose: (() => void)
+}
+
+const Transition = React.forwardRef(function Transition(
+  props: TransitionProps & { children?: React.ReactElement<any, any> },
+  ref: React.Ref<unknown>,
+) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+export default function AnalysisRunnerDialog({participants, open, onClose}: CancelAnalysisDialogProp) {
+    
+    const [state, setState] = useState({
+        checkedSNP: true,
+        checkedINDEL: true,
+        checkedDenovo: true,
+        checkedTranscripts: false,
+        checkedSynonynmous: true,
+        checkedSV: false,
+    });
+    const {checkedSNP, checkedINDEL, checkedDenovo, checkedTranscripts, checkedSynonynmous, checkedSV} = state;
+    const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setState({ ...state, [event.target.name]: event.target.checked });
+    };
+  
+    return (
+        <Dialog
+            open={open}
+            TransitionComponent={Transition}
+            keepMounted
+            onClose={onClose}
+            aria-labelledby="alert-dialog-slide-title"
+            aria-describedby="alert-dialog-slide-description"
+            maxWidth='md' 
+            fullWidth={true}
+        >
+            <DialogTitle id="alert-dialog-slide-title">
+                {"Run analysis"}
```

Comment:
Text nodes can just be written out inline, no need for `{""}`. This happens multiple times in this file.